UI/SidePanel.cpp

```diff
@@ -487,6 +488,10 @@ class SidePanel::PlanetPanel : public GG::Control {
       * returns the id of the clicked planet */
     mutable boost::signals2::signal<void (int)> LeftClickedSignal;
 
+    /** emitted when the planet is left double clicked by the user.
```

Comment:
LDCS is emitted by the PlanetPanel, PLDCS is emitted by the PlanetPanelContainer. This is how it is handled with (single) left as well as right-clicking. My understanding is that the signal gets passed down from PlanetPanel to PlanetPanelContainer to SidePanel, so that every class is notified of the click-event.